api/tests/unit/script/delete-user_test.js

```
@@ -0,0 +1,398 @@
+
+      it('should return the correct query', () => {
+        // arrange
+        const userId = 213;
+        // act
+        const query = subject.count_certifications_from_user_id(userId);
+        // assert
+        expect(query).to.equal(`SELECT COUNT(*) FROM "certification-courses" WHERE "userId" = '${userId}'`);
+      });
+
+    });
+
+    describe('#get_user_id_from_email', () => {
+      it('should return the correct query', () => {
+        // arrange
+        const email = 'jean.paul@pix.fr';
+        // act
+        const query = subject.get_user_id_from_email(email);
+        // assert
+        expect(query).to.equal(`SELECT id FROM users WHERE email = '${email}'`);
+      });
+    });
+
+    describe('#find_assessment_ids_from_user_id', () => {
+      it('should return the correct query', () => {
+        // arrange
+        const user_id = 123;
+        // act
+        const query = subject.find_assessment_ids_from_user_id(user_id);
+        // assert
+        expect(query).to.equal(`SELECT id FROM assessments WHERE "userId" = '${user_id}'`);
+      });
+    });
+
+    describe('#delete_feedbacks_from_assessment_ids', () => {
+      it('should return the correct query', () => {
+        // arrange
+        const assessment_ids = [123];
+        // act
+        const query = subject.delete_feedbacks_from_assessment_ids(assessment_ids);
+        // assert
+        expect(query).to.equal('DELETE FROM feedbacks WHERE "assessmentId" IN (123)');
+      });
+
+      it('should return the correct query with comma as separator when many assessment ids', () => {
+        // arrange
+        const assessment_ids = [123, 456];
+        // act
+        const query = subject.delete_feedbacks_from_assessment_ids(assessment_ids);
+        // assert
+        expect(query).to.equal('DELETE FROM feedbacks WHERE "assessmentId" IN (123,456)');
+      });
+
+      it('should return neutral query when assessmentIds is an empty array', () => {
```

Le test attend que l'on renvoie une erreur, ce qui me parait différent d'une requête neutre si je regarde le test seul. 
"Should throw an error when ... " ?

---

Ce commentaire peut se retrouver sur les autres tests qui lèvent une erreur